storybook/stories/Examples/Pie/PieWithNeedle.stories.tsx

```diff
@@ -42,35 +39,38 @@ const needle = (value: number, data: any[], cx: number, cy: number, iR: number,
 
   return [
     <circle cx={x0} cy={y0} r={r} fill={color} stroke="none" />,
-    <path d={`M${xba},${yba}L${xbb},${ybb},L${xp},${yp},L${xba},${yba}`} stroke="#none" fill={color} />,
+    <path
+      d={`M${x0},${y0}L${xbb + 65},${ybb - 65},L${xp},${yp},L${xba},${yba}`}
+      strokeWidth={2}
+      stroke={color}
+      fill={color}
+    />,
```

Comment:
```suggestion
    <path d={`M${x0},${y0}L${xbb + 65},${ybb - 65},L${xp}`} strokeWidth={2} stroke={color} fill={color} />,
```

Reply:
<img width="275" alt="Screenshot 2023-02-12 at 22 26 58" src="https://user-images.githubusercontent.com/27995606/218338172-a05e668d-ef52-403f-81f5-581e0cb47ac4.png">

This would work, and looks as such. 
Note you would have to remove unused variables.

Reply:
Ah yeah that looks better, I'm bad at SVG math